### PR TITLE
Consul 1.8.3 ubi8 image

### PR DESCRIPTION
**Pull request by snehakpersistent**
Updated Redhat ubi8 image for Consul 1.8.3

### CODE REVIEW

**Comment by seth-priya** [comment]
@kandarpamalipeddi please review

File: c/consul/Dockerfile/1.8.3-ubi8/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Building Consul
+```
+docker build -t consul .
+docker run -it consul:latest
+```
+
+# Deploying Consul on Openshift
+```
+$ oc new-project consul
+--> Now using project "consul" on server "https://api.sneha-ppc.cp.fyre.ibm.com:6443".
```
Comment (kandarpamalipeddi) [inline c/consul/Dockerfile/1.8.3-ubi8/README.md:10]: Can you replace the server address with **https://api-<cluster-address>:6443** ?
As no one can access the address as you are not providing username & password.

**Comment by kandarpamalipeddi** [comment]
Other than above comment,
Everything looks good.

File: c/consul/Dockerfile/1.8.3-ubi8/README.md (hunk 1 above)
Comment (snehakpersistent) [reply c/consul/Dockerfile/1.8.3-ubi8/README.md]: Updated.

**Comment by snehakpersistent** [comment]
@kandarpamalipeddi can we proceed to merge this?

**Review by kandarpamalipeddi — APPROVED** [review]

**Comment by kandarpamalipeddi** [comment]
@seth-priya 
LGTM

**Pull request merged**